mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -320,5 +339,88 @@ def _download_file(self, remote_file_path, local_path):
         read_credentials = self._get_read_credentials(self.run_id, run_relative_remote_file_path)
         self._download_from_cloud(read_credentials.credentials, local_path)
 
+    def download_artifacts(self, artifact_path, dst_path=None):
+        """
+        Parallelized implementation of `download_artifacts` for Databricks.
+        """
+
+        def download_file(fullpath):
```

Comment:
prefer to pass dst_path as an argument explicitly.

fullpath is confusing. Perhaps src_file for better readability. ditto below for dir_path -> src_dir etc.